.github/ISSUE_TEMPLATE/3-runtime-issue.yml

```diff
@@ -0,0 +1,26 @@
+name: Runtime Issue
+description: A crash is happening while your application is running
+labels: [ bug ]
+body:
+  - type: input
+    attributes:
+      label: SQLDelight Version
+    validations:
+      required: true
+  - type: input
+    attributes:
+      label: Application Operating System
+      placeholder: Android, iOS
```

Comment:
There could also be jvm or other OS targets from native

Reply:
The placeholder is just for example inputs, but I added `JVM` and `Native` as example inputs just to show that SQLDelight works also works beyond mobile.